request_left_downloader signal (#4303)
scrapy/scrapy: docs/topics/signals.rst

    :param spider: the spider that yielded the request
    :type spider: :class:`~scrapy.spiders.Spider` object

request_left_downloader

> ---------------------------

Reviewer: ```suggestion
-----------------------
```

I’m surprised the documentation build does not fail :slightly_smiling_face:

.. signal:: request_left_downloader
.. function:: request_left_downloader(request, spider)

>     Sent when a :class:`~scrapy.http.Request` left downloader even in case of

Reviewer 2: ```suggestion
    Sent when a :class:`~scrapy.http.Request` leaves the downloader even in case of
```

>     Sent when a :class:`~scrapy.http.Request` leaves the downloader even in case of

Reviewer: :lipstick: 

```suggestion
    Sent when a :class:`~scrapy.http.Request` leaves the downloader, even in case of
```

    failure.

>     The signal does not support returning deferreds from their handlers.

Reviewer 2: ```suggestion
    This signal does not support returning deferreds from its handlers.
```
I'm hopping to get the other occurrences of "their" updated in #4205

Author: It is done in accord with description of other signals, e.g https://github.com/scrapy/scrapy/blame/master/docs/topics/signals.rst#L257

Reviewer 2: I'm no native English speaker, but I believe "this" sounds better than "the". That's probably subjective though.
However I'm pretty sure "their" is incorrect, since we're talking about one signal, not many.

>     :param request: the request that reached downloader

Reviewer: ```suggestion
    :param request: the request that reached the downloader
```
